main.py

```diff
@@ -690,6 +719,13 @@ def parse_args(self) -> None:
             help="First submit button id to use in auth apps",
             required=False,
         )
+        parser.add_argument(
+            "--authSubmitAction",
+            help="Submit action to perform on form filled, click or submit",
+            type=str,
+            nargs="*",
```

Comment:
Should this be nargs? I think that's going to allow the user to specify multiple times & give you an array value.